markdown/the_photomode_wishlist.md

```diff
@@ -3,7 +3,7 @@
 <p><i>Alan Wake II</i>, shot by seymourglass</p>
 </div>
 
-This article is aimed at developers who are curious about what the "perfect" photomode looks like to players. Features cost resources so this list is organized by priority.
+This article is aimed at developers who are curious about what the "perfect" photomode looks like to players. Features cost resources, and the cost of implementing the ones listed in here can vary a lot, so this list is organized by priority.
```

Comment:
```suggestion
This article is aimed at developers who are curious about what the "perfect" photomode looks like to players. Features take resources, and the effort of implementing the ones listed in here can vary, so this list is organized by priority.
```

I tried to avoid referencing the "cost" in terms of money and put it in terms of effort — seems like it might be an easier sell that way.